addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1550,6 +1551,9 @@ const Wysiwyg = Widget.extend({
                         // Unstash the mutations now that the color is picked.
                         this.odooEditor.historyUnstash();
                         this._processAndApplyColor(eventName, ev.data.color);
+                        if (this.odooEditor.deselectTable() && hasValidSelection(this.odooEditor.editable)) {
```

Comment:
This is the colorpicker. Why should the colorpicker be concerned by deselecting the table ?

Reply:
That's the workaround we had decided to do so that we don't have the blue overlay of the selection hiding the selected color when we're done.

EDIT: Aaaah _that_ was why I had that fix in the previous commit! It doesn't work without it.